common/wuanalysis/anawu.hpp

```diff
@@ -20,13 +20,17 @@
 
 #include "anacommon.hpp"
 
-struct WuAnalyseOptions
+typedef enum { wuOptionFirst=0, minInterestingTime=0, minInterestingCost, skewThreshold } WuOptions ;
+
+interface IAnalyserOptions
```

Comment:
actually I'm not sure that this interface lives in this file either - it isn't part of the analyser public interface to the outside world.